src/resmom/requests.c

```diff
@@ -1863,7 +1863,6 @@ terminate_job(job *pjob, int internal)
 		}
 		if (kill_job(pjob, s) == 0) {
 			/* no processes around, force into exiting */
```

Comment:
Should you remove this comment? I don't think it makes much sense now

Reply:
I think the comment still fits since we are setting exiting_tasks=1 which will cause the rest of the job exit to take place.

Reply:
`exiting_tasks` just means we will eventually call `scan_for_exiting()`, right? I dunno what we're `force into exiting`

Reply:
updated the comment